[NOTASK]: Update styling of Home > Success Stories

eunicey opened the pull request:
*I made sure to not change any of the updates made in [[NOTASK]: unify gutter spacing, header spacing](https://github.com/poulami-saha/Global-Code-Queens-MyTeam-Website/pull/39/files#top) 
~~with the exception of one (see comments) which can be easily added back if necessary.~~

Updated styling of Success Stories
- spacing between elements especially at desktop width
- size of image
- font size of author name

BEFORE:
![BEFORE](https://github.com/user-attachments/assets/00ce4d36-3450-4aaf-a74f-59ea454dab5f)
AFTER:
![AFTER](https://github.com/user-attachments/assets/9738f866-473d-4bb8-9f90-561bbc9d59cf)

eunicey committed:
[NOTASK]: Update styling of Home > Success Stories

eunicey commented on src/components/HomeContainer/SuccessStories/SuccessStories.module.css:
```diff
@@ -78,7 +79,6 @@
 @media (min-width: 960px) {
   .wrapper {
     padding-block: 140px;
-    max-width: 1148px;
```
This is the only edit to what @joanwolk added in the previous PR. It includes the 165px L/R padding so the content width is quite narrow for desktop.

eunicey replied:
Ignore my previous comment. In my latest commit, I've added max-width back but made the box-sizing to content-box.

eunicey committed:
Return max-width but add border-box with wrapper

eunicey committed:
Correct box-sizing

joanwolk commented:
Using content-box breaks displays for small desktops, where the stories overflow the content area and also the viewport:
<img width="997" alt="Screenshot 2024-08-01 at 9 53 48 AM" src="https://github.com/user-attachments/assets/753c6b16-d8c9-4d4b-b030-0e1b19a6d3be">

Using border-box and putting the max-width on `.contents` instead of `.wrapper` will get things _mostly_ working, although the names will still misalign at some sizes. The solution for that is some work with `display: grid` I think, although I'm having some trouble executing the thing that should work.

eunicey committed:
Move max-width from .wrapper to .contents

eunicey commented:
@joanwolk - Thanks for the suggestion. I made the change you recommended for the max-width  and box-sizing. I won't have the time today to explore the display grid solution though.

joanwolk committed:
remove duplicate styles - layout styles already set border-box on all elements

joanwolk committed:
use grid to lock alignment on success stories
- .contents grid sets tidy columns, and aligns child items to the end of the block
- .quote is fixed to full height and uses rows of 1fr to set height of child blocks smartly
- .quote > :first-child sets the classless first child of .quote to align at the start of the grid instead of the end, so the extra whitespace is after the quotation and before the name

joanwolk commented:
I solved my grid issue!
- `.contents` grid sets tidy columns, and aligns child items to the end of the block
- `.quote` is fixed to full height and uses rows of 1fr to set height of child blocks smartly
- .`quote > :first-child` sets the classless first child of `.quote` to align at the start of the grid instead of the end, so the extra whitespace is after the quotation and before the name (and I wouldn't have to change a snapshot if one were present, as I would if I added a class)

poulami-saha commented on src/components/HomeContainer/SuccessStories/SuccessStories.module.css:
```diff
@@ -29,14 +28,14 @@
 .image {
-  width: 60px;
-  height: 60px;
+  width: 66px;
```
The width and height of each image is 62px across all screen size, as per the design, can we have a quick look at it?

eunicey replied:
@joanwolk - I included the 2px border which is not accounted for in the design when you click on the image size. I did the same for the image on the profile cards.
